packages/console/src/hooks/use-connector-groups.ts

```diff
@@ -8,7 +8,7 @@ import type { ConnectorGroup } from '@/types/connector';
 
 // Group connectors by target
 const useConnectorGroups = () => {
-  const { data, ...rest } = useSWR<ConnectorResponse[], RequestError>('/api/connectors');
+  const { data, ...rest } = useSWR<ConnectorFactoryResponse[], RequestError>('/api/connectors');
```

Comment:
For normal response, I'll suggest keeping the original "ConnectorResponse", for the front-end doesn't care about "factory"

Reply:
Undo the changes